index.html

```diff
@@ -203,9 +203,9 @@ <h3 class="work_title">Skills</h3>
     <section id="form_section">
         <h1 class="primary-font-style contact_form_text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
         <form class="input_form">
```

Comment:
Good job, but please make sure to add the `method` attribute with a proper value and the `action` attribute with the link to the [Formspree](https://formspree.io/html) form.